def cleaning_exceptions(e):
	exception_count+=1
	print(e)
	print(exception_count)